lib/registrant.js

```diff
@@ -254,6 +241,52 @@ Registrant.prototype.setThingValid = function(identity, isValid) {
 };
 
 /**
+ * Add new Thing schema to the Registry on the blockchain.
+ * @function
+ * @param {Object} schema that will be added.
+ * @param {Object} testThingData to try to build Thing with new schema.
+ * @returns {Promise} that resolves with transaction hash of the operation.
+ * @note Schema object is of template: { name: '', description: '', definition: '' };
+ * @note Sends transaction, costs Ether.
+ */
+Registrant.prototype.createSchema = function(schema, testThingData) {
```

Comment:
Not sure about `testThingData`. It's nice as separate feature, but may be a bit misleading.
Also in most cases providing empty object as testThingData will work just fine, since usually all fields are optional. Even providing object with no intersecting keys with proposed schema keys will work fine as well.
Or maybe you have some reasoning behind it?
I would suggest to move it from here to separate function that will allow to test proposed schema against thing before actually creating one.